installutils/helminstall/client.go

```diff
@@ -190,7 +191,7 @@ func (d *defaultHelmClient) ReleaseExists(namespace, releaseName string) (bool,
 
 // Build a Helm EnvSettings struct
 // basically, abstracted cli.New() into our own function call because of the weirdness described in the big comment below
-func NewCLISettings(namespace string) *cli.EnvSettings {
+func NewCLISettings(kubeConfig, kubeContext, namespace string) *cli.EnvSettings {
 	// The installation namespace is expressed as a "config override" in the Helm internals
 	// It's normally set by the --namespace flag when invoking the Helm binary, which ends up
```

Comment:
can we generalize this comment a bit? Because now it applies to more than just the one value that comes in over an env var